scripts/test_antlr_grammar.sh

```diff
@@ -39,19 +39,15 @@ prepare_workdir()
 prepare_workdir
 download_antlr4
 
-if [[ ! -f "${WORKDIR}/target/SolidityParser.class" ]] || \
-    [ "${GRAMMAR_FILE}" -nt "${WORKDIR}/target/SolidityParser.class" ]
-then
```

Comment:
Not regenerating the parser, even if the source files is more cumbersome than useful I'd say.